src/ast/mod.rs

```diff
@@ -1524,6 +1525,20 @@ pub enum Statement {
         /// Optional parameters.
         params: CreateFunctionBody,
     },
+    /// ```sql
+    /// CREATE STAGE
+    /// ```
```

Comment:
could you add a link to the docs? https://docs.snowflake.com/en/sql-reference/sql/create-stage